consensus/polybft/command/polybftsecrets/secrets_init.go

```diff
@@ -0,0 +1,41 @@
+package polybftsecrets
```

Comment:
This functionality should be integrated with the existing `secrets init` command, and not copied over and modified

Reply:
It was an intentional decision to not integrate it with `secrets init` yet. We expect to follow better guidelines when integrating with `edge` code that it is already in production and this PR was meant to be a WIP.